### DISCUSSION TITLE
Does fvm support flutter for arm64 linux ? If yes how ?

### DISCUSSION THREAD

**Discussion by vednig** [leoafarias/fvm#755] [Q&A]
As the question states...

**Comment by vednig** [comment]
@leoafarias any suggestions?

**Reply by leoafarias** [reply]
Does this solve the problem?

https://github.com/leoafarias/fvm/pull/747?

**Reply by vednig** [reply]
I will look into it.

**Comment by AngeloAvv** [comment]
Hey @leoafarias , do you think we can also release a docker image which support arm64 linux?
I created a [PR](https://github.com/leoafarias/fvm/pull/845) which is also linked to an [older issue](https://github.com/leoafarias/fvm/issues/762) I opened almost one year ago.
